src/components/main/Main.jsx

```diff
@@ -0,0 +1,61 @@
+import "./main.css";
+import { CategoryCard } from "./components/CatergoryCard";
+import { trendingProducts } from "../../data/data.products";
+import { ProductCard } from "../productCard/productCard";
+function Main() {
+  return (
+    <div>
+      <section className="hero d-flex align-center">
+        <div className="banner">
+          <span className="banner-text">Gamers don't die, They respawn.</span>
+          <p className="banner-text mb-m">Buy Games at Flat 40% OFF</p>
+          <a href="/" className="btn btn-secondary fs-m">
+            {" "}
+            Shop Now
+          </a>
+        </div>
+      </section>
+
+      <section className="featured-categories">
+        <h1 className="fs-xl text-align-center">Featured Categories</h1>
+        <div className="title-underline"></div>
+        <div className="grid">
+          <CategoryCard />
```

Comment:
Nitpicking: Instead of putting all categories inside the category card you could have just created a component for one card and done the iteration here because let's suppose you need the same card component somewhere else but with just 2 items it will not be extensible.

Reply:
True, but this is only for the categories and this will not be used elsewhere. If I need to use this somewhere else, I'll refactor the code. Only for this reason I've put this inside a components folder inside main.